bson/src/main/org/bson/codecs/pojo/annotations/Property.java

```diff
@@ -16,30 +16,32 @@
 
 package org.bson.codecs.pojo.annotations;
 
+import org.bson.codecs.pojo.PropertyModel;
+
 import java.lang.annotation.Documented;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
 /**
- * An annotation that configures a field.
+ * An annotation that configures a property.
  *
  * <p>Note: Requires the {@link org.bson.codecs.pojo.Conventions#ANNOTATION_CONVENTION}</p>
  *
  * @since 3.5
  * @see org.bson.codecs.pojo.Conventions#ANNOTATION_CONVENTION
  */
 @Documented
-@Target(ElementType.FIELD)
+@Target({ElementType.METHOD, ElementType.FIELD, ElementType.PARAMETER})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Property {
```

Comment:
No, it was still open / tbc. I like the Bson prefix - so have updated and think it looks reasonable.